pom.xml

```diff
@@ -64,31 +64,31 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
```

Comment:
`ov-platform` uses `1.7.5` which is vulnerable too, propose to update create issue

Reply:
Already done [Sec-218320](https://trackor.onevizion.com/form/ConfigAppForm.do?id=100107144075&ttid=10009161)